### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by MasterRadule**

### CODE REVIEW

**Review by LazarJovic — APPROVED** [review]

File: Literary-Association-Application/src/main/java/goveed20/LiteraryAssociationApplication/services/PdfService.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 @Service
 public class PdfService {
-    private static final String writingsFolder = "Literary-Association-Application/src/main/resources/writings/";
+    private static final String booksFolder = "Literary-Association-Application/src/main/resources/books/";
```
Comment (draganculibrk9) [inline Literary-Association-Application/src/main/java/goveed20/LiteraryAssociationApplication/services/PdfService.java:18]: Remove this change

File: Literary-Association-Application/src/main/java/goveed20/LiteraryAssociationApplication/services/BookService.java [hunk 2]
```diff
@@ -157,4 +157,18 @@ public ResponseEntity downloadBook(String bookTitle) throws Exception {
...
     }
+
+    private List<String> getBookTitles(BaseUser reader) {
+        List<String> bookTitles = new ArrayList<>();
+
+        if (SecurityContextHolder.getContext().getAuthentication().getAuthorities().contains(UserRole.READER)) {
+            reader.getTransactions().forEach(t -> {
+                if (t.getStatus().equals(TransactionStatus.COMPLETED) && !(t instanceof MembershipTransaction)) {
+                    t.getInvoice().getInvoiceItems().forEach(it -> bookTitles.add(it.getName()));
+                }
```
Comment (draganculibrk9) [inline Literary-Association-Application/src/main/java/goveed20/LiteraryAssociationApplication/services/BookService.java:168]: Use stream filter and distinct for titles

File: Literary-Association-Frontend/src/reducers/bookReducer.js [hunk 3]
```diff
@@ -22,22 +27,28 @@ export const setBook = (id) => {
     }
 }
 
-const reducer = (state = { list: [], shown: null }, action) => {
+const reducer = (state = {list: [], shown: null}, action) => {
```
Comment (draganculibrk9) [inline Literary-Association-Frontend/src/reducers/bookReducer.js:30]: Add myBooks boolean to state and set it to default false value

**Review by draganculibrk9 — CHANGES_REQUESTED** [review]
Make requested changes and test affected functionalities for roles READER and WRITER

**Review by draganculibrk9 — APPROVED** [review]

**Pull request merged**